cves/kernel/CVE-2013-0290.yml

```
@@ -147,10 +148,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    The vulnerability was discovered by a developer, 
+    Tommi Rantala <tt.rantala@gmail.com>, testing the code with a fuzzer
+  automated: false
+  contest: false
+  developer: true
```

This threw me off too in my work. I think this is false. I can't seem to find information on Tommi Rantala being a Linux Kernel developer or Google developer. I think he was just a contributor. His email is just standard gmail. Confirm before you make the change.

```suggestion
  developer: false
```

Edit: I see it was signed off by Eric Dumazet, a Google developer. So I'm not sure what to do for this one...